.github/workflows/ci-actions.yml

```diff
@@ -154,7 +183,8 @@ jobs:
         shell: bash
         run: tar -xzf maven-repo.tgz -C ~
       - name: Build with Maven
-        run: eval mvn $JVM_TEST_MAVEN_OPTS install -pl !integration-tests/gradle -pl !integration-tests/maven -pl !integration-tests/devtools ${{ matrix.java.maven_args}}
```

Comment:
I had some nasty formatting/escaping problems with `eval`, especially with `-Dgib.disableIfBranchRegex='master|\d+\.\d+|.*backport.*'` (see further up).
AFAICT `eval` ist not required so I simply removed it from every `mvn` call.